rest/route/host_test.go

```diff
@@ -863,6 +865,51 @@ func TestClearHostsHandler(t *testing.T) {
 	assert.Equal(t, res.TerminatedVolumes[0], "v1")
 }
 
+func TestRemoveAdminHandler(t *testing.T) {
+	assert.NoError(t, db.ClearCollections(dbModel.ProjectRefCollection))
+	projectRef0 := &dbModel.ProjectRef{
+		Owner:     "mongodb",
+		Repo:      "test_repo0",
+		Branch:    "master",
+		RepoKind:  "github",
+		Enabled:   true,
+		BatchTime: 10,
+		Id:        "test0",
+		Admins:    []string{"user1", "user0"},
+	}
+	projectRef1 := &dbModel.ProjectRef{
+		Owner:     "mongodb",
+		Repo:      "test_repo1",
+		Branch:    "master",
+		RepoKind:  "github",
+		Enabled:   true,
+		BatchTime: 10,
+		Id:        "test1",
+		Admins:    []string{"user1", "user2"},
+	}
+
+	assert.NoError(t, projectRef0.Insert())
+	assert.NoError(t, projectRef1.Insert())
+
+	offboardedUser := "user0"
+	handler := offboardUserHandler{
+		sc:     &data.DBConnector{},
+		dryRun: false,
+		env:    evergreen.GetEnvironment(),
+		user:   offboardedUser,
+	}
+	handler.env.SetUserManager(serviceutil.MockUserManager{})
+
+	resp := handler.Run(gimlet.AttachUser(context.Background(), &user.DBUser{Id: "root"}))
+	require.Equal(t, http.StatusOK, resp.Status())
+
+	projectRefs, err := dbModel.FindAllProjectRefs()
```

Comment:
You may also want to verify the length of projectRefs is what you expect, and you're not just getting no results and skipping the 908 loop